libvncclient/rfbproto.c

```diff
@@ -1392,7 +1394,11 @@ SendFramebufferUpdateRequest(rfbClient* client, int x, int y, int w, int h, rfbB
   rfbFramebufferUpdateRequestMsg fur;
 
   if (!SupportsClient2Server(client, rfbFramebufferUpdateRequest)) return TRUE;
-  
+  rfbClientLog("FramebufferUpdateRequest: %d,%d,%d,%d\n", x,y,w,h);
```

Comment:
This probably means a lot of messages. Please remove.

Reply:
I agree with @bk138 this introduce a lot of log messages, thus it should be removed